api/api/configuration.py

```diff
@@ -24,9 +24,9 @@
     "host": "0.0.0.0",
     "port": 55000,
     "behind_proxy_server": False,
+    "auth_token_exp_timeout": 36000,
```

Comment:
```suggestion
    "auth_token_exp_timeout": 900,
```

Reply:
As this is not ready for production yet, I rather have this long timeout for testing.